packages/electron-app/src/database/tradeService.js

```
@@ -1,41 +1,147 @@
+    const posTx = db.prepare(`
+      SELECT MAX(t.datetime) as max_datetime 
+      FROM transactions t
+      JOIN trades tr ON t.trade_id = tr.trade_id
+      WHERE tr.instrument_ticker = @instrument_ticker AND tr.asset_class = @asset_class AND tr.exchange = @exchange AND tr.trade_direction = @trade_direction
+    `).get({
+        instrument_ticker: tradeInfoForLatest.instrument_ticker, 
+        asset_class: tradeInfoForLatest.asset_class, 
+        exchange: tradeInfoForLatest.exchange, 
+        trade_direction: tradeInfoForLatest.trade_direction
+    });
     latestTradeDatetime = posTx && posTx.max_datetime ? posTx.max_datetime : null;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)